synapse/cortex.py

```diff
@@ -2206,6 +2206,7 @@ async def _initCoreViews(self):
             vdef = {
                 'layers': (layriden,),
                 'worldreadable': True,
+                'creator': self.auth.rootuser.iden
```

Comment:
this is already stamped into the vdef during addView()

Reply:
Aha, confirmed the issue is OBE then :)